app/models/concerns/account/interactions.rb

```diff
@@ -80,8 +80,8 @@ def follow_mapping(query, field)
       has_many :passive_relationships, foreign_key: 'target_account_id', inverse_of: :target_account
     end
 
-    has_many :following, -> { order('follows.id desc') }, through: :active_relationships,  source: :target_account
-    has_many :followers, -> { order('follows.id desc') }, through: :passive_relationships, source: :account
+    has_many :following, -> { Follow.recent }, through: :active_relationships,  source: :target_account
```

Comment:
This is better, but we should also work to remove these orders from the associations IN THEIR ENTIRETY!!!